doc/source/usage.rst

```diff
@@ -54,15 +54,30 @@ The syntax to run a command depends on how the package is installed:
 
         ksrates [OPTIONS] COMMAND [ARGS]
 
-*   Singularity container::
+*   Singularity container:
+
+        Open an interactive container where to launch commands with the syntax indicated in the local installation above::
+
+            singularity shell docker://vibpsb/ksrates
+
+        Or launch a single command through the container::
 
-        singularity exec docker://vibpsb/ksrates ksrates [OPTIONS] COMMAND [ARGS]
+            singularity exec docker://vibpsb/ksrates ksrates [OPTIONS] COMMAND [ARGS]
+
+        .. note::
+            WSL2 users need option ``-B /mnt/c/Users/your_username`` to mount the Windows home directory in the container.
```

Comment:
Can we assume it's always "c/Users/your_username" and that exists or shall we note it's an example?